CONTRIBUTING.md

```diff
@@ -12,6 +12,9 @@ When opening new issues or commenting on existing issues on this repository
 please make sure discussions are related to concrete technical issues with the
 Node.js software.
 
+Go to https://github.com/nodejs/node/issues/new and click the 'New Issue' button
```

Comment:
I'm definitely -1 on including the bare URL. I'm not sure any of this paragraph is needed at all. Do we have any reason to believe that people don't know how to open issues on GitHub and are looking for the information in the CONTRIBUTING.md? Most people only see the CONTRIBUTING.md when they are already opening a PR or issue because that's when GitHub shows a link to it.